docs/content/latest/cdc/cdc_documentation_v1.md

```diff
@@ -0,0 +1,187 @@
+---
```

Comment:
Needs front matter, and this is the Explore page — probably needs content broken out for reference, as well.